src/coreclr/src/vm/classnames.h

```diff
@@ -82,6 +82,7 @@
 #define g_ReflectionReflectItfName "System.Reflection.IReflect"
 #define g_RuntimeArgumentHandleName      "RuntimeArgumentHandle"
 #define g_RuntimeFieldHandleClassName    "System.RuntimeFieldHandle"
+#define g_RuntimeFieldHandleInternalName        "RuntimeFieldHandleInternal"
```

Comment:
Where was this previously?

Reply:
This name was not used previously in native code. Now we use it to explicitly normalize this type as a `nint` the old way, like `RuntimeMethodHandleInternal` and family. I added this initially to fix FCalls that were failing due to this missing. On the native side, we use `MethodDesc*` and `FieldDesc*`, but on the managed side we use the `RuntimeMethodHandleInternal` and `RuntimeFieldHandleInternal` structs. The old way of normalization basically tells the JIT that the type is a `nint` instead of handling the struct type directly. Since the native side uses pointers, I figured that we should keep using this style of normalization for this family of types to ensure that we don't have different calling convention issues in the future.